### PR TITLE
[12.0][MIG] project_risk : Migration to 12.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration of project_risk to 12.0

@Tecnativa TT22496

### CODE REVIEW

File: project_risk/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    'license': 'AGPL-3',
+    'website': 'https://github.com/OCA/project',
+    'category': 'Project Management',
+    'version': '12.0.1.0.0',
+    'depends': [
+        'project'
+    ],
+    'data': [
+        'security/ir_model_access.xml',
+
```
Comment (emagdalenaC2i) [inline project_risk/__manifest__.py]: Delete extra line

File: project_risk/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        'data/project_risk_response_category_data.xml',
+        'data/project_risk_category_data.xml',
+
+        'views/project_risk_response_category_view.xml',
+        'views/project_risk_category_view.xml',
+        'views/project_risk_view.xml',
+        'views/project_project_view.xml',
+
```
Comment (emagdalenaC2i) [inline project_risk/__manifest__.py]: Delete extra line

File: project_risk/menuitems.xml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (emagdalenaC2i) [inline project_risk/menuitems.xml]: Why use a separate file instead of including this code in project/project_risk/views/project_risk_view.xml ?

**Review by emagdalenaC2i — APPROVED** [review]
Code LGTM I didn't tested in runbot

File: project_risk/__manifest__.py (hunk 1 above)
Comment (manuelcalerosolis) [reply project_risk/__manifest__.py]: Deleted

File: project_risk/__manifest__.py (hunk 2 above)
Comment (manuelcalerosolis) [reply project_risk/__manifest__.py]: Deleted

File: project_risk/menuitems.xml (hunk 3 above)
Comment (manuelcalerosolis) [reply project_risk/menuitems.xml]: The previous version 11.0 locate the file here, now move the file to views directory.

Thanks

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**